src/abstract_charm.py

```diff
@@ -67,6 +67,9 @@ def _subordinate_relation_endpoint_names(self) -> typing.Optional[typing.Iterabl
         Does NOT include relations where charm is principal
         """
 
+    def is_exposed(self, relation=None) -> bool:
+        return self._database_provides.is_exposed(relation)
```

Comment:
can the relation in database_provides be made a kwarg? i realize there can be multiple relations with clients for the k8s charm, but in the vm, there's only one, and so this should be an optional kwarg in my opinion

Reply:
@shayancanonical can I close this comment then?

Reply:
yes, please close this comment. i will reflect the change in the vm charm as i pick up the implementation there for [DPE-3689](https://warthogs.atlassian.net/browse/DPE-3689)